webclient/static/js/dz.js

```diff
@@ -280,4 +303,26 @@ window.onload = function () {
         newDropzone.emit("complete", mockFile);
     });
 
+    //zablokuje tlačítka mazání
+    function disableRemoving(lock) {
```

Comment:
Chápu dobře, že ta funkce se jmenuje `disableRemoving`, ale ve skutečnosti může tlačítko (v závislosti na parametru lock) povolit i zakázat? Pokud ano, tak bych spíše přidal funkce `disableRemoving` a `enableRemoving`, aby to bylo čitelné.